### PR TITLE
372: Upgrade flutter

### PR DESCRIPTION
**Pull request by michael-markl**
I tested all features that are in production on Android. I did not test anything regarding "Ausweisen" / "Beantragen".
- [x] iOS still needs to be tested @maxammann  / @RoAnBu

### CODE REVIEW

**Review by elKei24 — APPROVED** [review]
Once again: did not test it, but code looks fine on the first glance. Someone should test it before the PR is merged, though.

File: frontend/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -25,15 +25,6 @@
...
               />
-            <!-- Displays an Android View that continues showing the launch screen
-                 Drawable until Flutter paints its first frame, then this splash
-                 screen fades out. A splash screen is useful to avoid any visual
-                 gap between the end of Android's launch screen and the painting of
-                 Flutter's first frame. -->
-            <meta-data
-              android:name="io.flutter.embedding.android.SplashScreenDrawable"
-              android:resource="@drawable/launch_background"
-              />
```
Comment (maxammann) [inline frontend/android/app/src/main/AndroidManifest.xml:36]: How did you come up with this change? Is this like in RN where there is a diff which should be applied to Flutter projects upon upgrading?

**Review by maxammann — COMMENTED** [review]
Looks good, testing now on iOS

File: frontend/android/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (Schedulaar) [reply frontend/android/app/src/main/AndroidManifest.xml:36]: And I think there was a deprecation notice in the console when building the app

**Comment by maxammann** [comment]
Had to do a few changes on iOS. Overall it worked good!

**Pull request merged**